packages/plugins/src/cli/deploy.ts

```diff
@@ -277,13 +326,65 @@ export const deploy = async (
     await sphinxContext.prompt(previewString)
   }
 
-  const { receipts } = await runEntireDeploymentProcess(
-    parsedConfig,
-    merkleTree,
+  const treeSigner = {
+    signer: await signer.getAddress(),
+    signature: await signMerkleRoot(merkleTree.root, signer),
+  }
+  const deployment: Deployment = {
+    id: 'only required on website',
+    multichainDeploymentId: 'only required on website',
+    projectId: 'only required on website',
```

Comment:
thoughts on making the types of these fields `string | undefined` or `string | null`? seems cleaner than saying 'only required on website' a bunch of times

Reply:
That would add a lot of complexity to the website because we'd have to include handling for cases where these fields are not defined since the type structure no longer enforces it. Given that, I think it's better to require them to be defined and just give them values that convey that they are not necessary for certain contexts.